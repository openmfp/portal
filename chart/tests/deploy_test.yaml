suite: deploy
templates:
  - deploy.yaml
tests:
  - it: matches the snapshot
    values:
      - ../test-values.yaml
    template: deploy.yaml
    asserts:
      - matchSnapshot: {}
