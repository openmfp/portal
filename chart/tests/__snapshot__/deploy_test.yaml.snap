matches the snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app: RELEASE-NAME
      name: RELEASE-NAME
      namespace: NAMESPACE
    spec:
      revisionHistoryLimit: 3
      selector:
        matchLabels:
          app: RELEASE-NAME
      strategy:
        rollingUpdate:
          maxSurge: 5
          maxUnavailable: 0
      template:
        metadata:
          labels:
            app: RELEASE-NAME
        spec:
          containers:
            - env:
                - name: OIDC_CLIENT_ID_PORTAL
                  value: 12345
                - name: OIDC_CLIENT_SECRET_PORTAL
                  valueFrom:
                    secretKeyRef:
                      key: secret
                      name: portal-client-secret-portal
                - name: AUTH_SERVER_URL_PORTAL
                  value: https://auth.example.com
                - name: TOKEN_URL_PORTAL
                  value: https://auth.example.com/oauth2/token
                - name: BASE_DOMAINS_PORTAL
                  value: portal.example.com,portal2.example.com
                - name: IDP_NAMES
                  value: portal
                - name: PORT
                  value: "8080"
                - name: REGION
                  value: null
                - name: IMAGE_TAG
                  value: "12345"
                - name: IMAGE_NAME
                  value: ghcr.io/openmfp/portal
                - name: COOKIE_DOMAIN
                  value: null
              image: ghcr.io/openmfp/portal:12345
              imagePullPolicy: IfNotPresent
              name: RELEASE-NAME
              ports:
                - containerPort: 8080
                  name: http
              resources:
                limits:
                  cpu: "1"
                  memory: 512Mi
                requests:
                  cpu: 40m
                  memory: 50Mi
              securityContext:
                readOnlyRootFilesystem: true
          securityContext:
            fsGroup: 2000
            runAsGroup: 3000
            runAsUser: 1000
          serviceAccountName: RELEASE-NAME
